knowledge_base/save_job_result_to_volume/databricks.yml

```diff
@@ -0,0 +1,12 @@
+bundle:
+  name: save_job_result_to_volume
+
+include:
+  - resources/*.yml
+
+workspace:
+  host: https://e2-dogfood.staging.cloud.databricks.com
+
+targets:
+  dev:
+    default: true
```

Comment:
`mode: development`

Reply:
Discussed offline, we'll skip `mode: development` here because of the existing gap where schemas support prefixing, but volumes don't. We intend to remove schema prefixing support in the future to make this consistent.